.github/workflows/pull-request.yml

```diff
@@ -75,18 +75,14 @@ jobs:
       run: docker stop nginx certbot frontend backend
     - name: Delete additon info
       run: docker system prune -f
-    - name: Set docker tag
-      run: echo "DOCKER_TAG_BACKEND=${GITHUB_REF##*/}.${GITHUB_RUN_NUMBER}" >> $GITHUB_ENV
-    - name: Check value
-      run: echo "$DOCKER_TAG_BACKEND"
     - name: Set up enviroments and Deploy
       env:
         INITIAL_ADMIN_EMAIL: ${{ secrets.INITIAL_ADMIN_EMAIL }}
         INITIAL_ADMIN_PASSWORD: ${{ secrets.INITIAL_ADMIN_PASSWORD }}
         JWTOPTIONS_SECRETKEY: ${{ secrets.JWTOPTIONS_SECRETKEY }}
         JWTOPTIONS_REFRESH_TOKEN_SECRETKEY: ${{ secrets.JWTOPTIONS_REFRESH_TOKEN_SECRETKEY }}
         DOCKER_TAG_FRONTEND: ${{ steps.frontend_tag.outputs.frontend_tag }}
-        DOCKER_TAG_BACKEND: backend:release-${{ env.DOCKER_TAG_BACKEND }}
+        DOCKER_TAG_BACKEND: ${{ steps.backend_tag.outputs.frontend_tag }}
```

Comment:
Sorry for leaving you comment about this one more time

So for front end and backend we will use same `steps.backend_tag.outputs.frontend_tag` ?

It is because only `frontend_tag` is present? 

Can we add similar one ? For back end specifically?
Like  `steps.backend_tag.outputs.backend_tag` ?